dbt_project.yml

```diff
@@ -21,16 +21,42 @@ vars:
 
 models:
   synthea_omop_etl:
-    +materialized: table
+    intermediate:
```

Comment:
i got these colors from https://forums.ohdsi.org/t/ohdsi-palette-for-plots/20120

sadly, the color for sources (green) cannot be changed. i pinged the feature request for this in dbt repo here: https://github.com/dbt-labs/dbt-docs/issues/26#issuecomment-2563086485